server/container/guice/mailet/src/test/java/org/apache/james/utils/FileSystemFixture.java

```diff
@@ -54,7 +54,7 @@ public InputStream getResource(String url) throws IOException {
 
         @Override
         public File getFile(String fileURL) throws FileNotFoundException {
-            return new File(ClassLoader.getSystemResource("recursive/included-jars").getFile());
```

Comment:
I guess the NPE comes from not having renamed the test resources folder to **recursive/extensions-jars**